verify.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+travis_tag=$1
```

Comment:
Running this before the deploy would be better than every time, yes. Then I wouldn't have to check the condition in the script. I just didn't know how to do that, I don't know what the deploy is actually doing.